Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -296,6 +311,7 @@ final class PostCampaignCheckoutViewController: UIViewController, MessageBannerV
         guard error == nil, status == .succeeded else {
           self.messageBannerViewController?
             .showBanner(with: .error, message: Strings.Something_went_wrong_please_try_again())
+          self.viewModel.inputs.checkoutTerminated()
```

Comment:
I don't like how many different places we're calling `checkoutTerminated` in this - it feels messy. But I can't think of a better way to implement this. Let me know if y'all have any ideas.

Reply:
You could write a wrapper method like 

```
checkoutTerminated(withErrorMessage message: String?) {
    self.messageBannerViewController?
            .showBanner(with: .error, message: message ?? Strings.Something_went_wrong_please_try_again())
    self.viewModel.inputs.checkoutTerminated()
}
```

which would consolidate two of these.

Reply:
That feels more confusing to me actually, since then the `checkoutTerminated` would sometimes be nested and sometimes not. I think I'm going to leave this as-is if you're okay with it. I also think it's clearer now that I got rid of the extraneous calls, so thanks for that call-out!

The other thing I tried was having the processing view tied directly to api calls the way the PledgeViewModel does, but it did disappear and reappear between linked calls, so I say that very much did not work.